packages/core-react/src/components/player/useSourceMimeTyped.tsx

```diff
@@ -56,6 +55,13 @@ export const useSourceMimeTyped = <TElement, TPoster>({
     onAssetStatusChange,
   });
 
+  const [isStreamOffline, setIsStreamOffline] = React.useState<boolean>(false);
+
+  React.useEffect(() => {
```

Comment:
We could use a `useMemo` here for simplicity - something like

```suggestion
const isStreamOffline = React.useMemo(() => playbackInfo?.type === 'live' && !playbackInfo?.meta?.live);
```

Reply:
Good point, it's just simpler this way